sdk/src/types/evaluation.d.ts

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable */
+/**
+ * This file was automatically generated by json-schema-to-typescript.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,
+ * and run json-schema-to-typescript to regenerate this file.
+ */
+
+export type EvaluationSource = EASEvaluation | IPFSEvaluation;
+
+/**
+ * Schema for evaluating Hypercerts accross different sources and evaluation types
+ */
+export interface HypercertEvaluationSchema {
+  creator: string;
```

Comment:
What goes in here? How are we encoding identities?

Reply:
The creator? I think it should be the signer address, I can make that explicit in documentation and validate on a valid address when parsing the evaluation

Reply:
pushed